src/app/common/switch-account/use-switch-account-modal-context.ts

```diff
@@ -0,0 +1,15 @@
+import { useOutletContext } from 'react-router-dom';
+
+import type { SwitchAccountOutletContext } from './switch-account';
+
+export function useSwitchAccountModal() {
```

Comment:
Mark and the design team had asked me to rename all `modal`/ `dialog` as `sheet` so maybe this should also be a `sheet`